test/runtests.jl

```diff
@@ -64,6 +66,15 @@ const GROUP = get(ENV, "GROUP", "")
 include("test_utils.jl")
 
 @testset "KernelFunctions" begin
+    if GROUP == "" || GROUP == "Aqua"
+        @testset "Aqua" begin
+            Aqua.test_all(KernelFunctions; ambiguities=false)
+            @testset "Method Ambiguity" begin
+                Aqua.test_ambiguities([KernelFunctions, Base, Core]; recursive=false)
+                # Avoid checking extra package ambiguity issues
+            end
```

Comment:
Based on https://github.com/JuliaTesting/Aqua.jl/issues/77 (and in particular Jameson's comments) I think this should be changed to
```suggestion
            # Ref https://github.com/JuliaTesting/Aqua.jl/issues/77
            Aqua.test_ambiguities(KernelFunctions; recursive=false)
```